docs/modules/general/pages/local-env.adoc

```diff
@@ -0,0 +1,26 @@
+= OneCX Local Environment
+
+== Overview
+With OneCX platform, every Application needs to adjusted so it can be integrated within the platform. For the best experience with Application development and integration, OneCX offers a https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository. It provides a consistent setup of the platform and Core Applications, with minimal actions and allows extention.
+
+== Setting Up OneCX and Core Applications
+Using https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository allows to:
+
+* set up all required services to run the OneCX platform
+* set up Shell and other Core Applications with preconfigured data
+* develop and test Applications already integrated with the platform
+* easily extend and overwrite the configuration of the platform and Applications
+
+=== Prerequisites
+It is recommended to get familiar with the README of the https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository to make sure the machine's configuration meets the expectations. It is important to follow it cautiously. Assuming all the steps have been completed, it should be possible to access the OneCX platform locally with preconfigured Shell and Core Applications.
```

Comment:
Only really minor thing: Maybe directly link to the README https://github.com/onecx/onecx-local-env?tab=readme-ov-file#onecx-local-environment-setup but up to you